manifests/a/Autodesk/ShotgridRV/2022.2.0/Autodesk.ShotgridRV.yaml

```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Autodesk.ShotgridRV
+PackageVersion: 2022.2.0

```

Comment:
```suggestion
PackageVersion: 22.2.0
```

Reply:
2022 is correct

Reply:
I posted the proof that it is not.
Complain to the developer.